ui/src/app/applications/components/application-sync-options/application-sync-options.tsx

```diff
@@ -6,6 +6,7 @@ import * as ReactForm from 'react-form';
 require('./application-sync-options.scss');
 
 export const REPLACE_WARNING = `The resources will be synced using 'kubectl replace/create' command that is a potentially destructive action and might cause resources recreation.`;
+export const FORCE_WARNING = `The resources will be synced using '--force' that is a potentially destructive action and some resources may result in inconsistency or data loss.`;
```

Comment:
Here is the description from kubectl:

```
--force=false: If true, immediately remove resources from API and bypass graceful deletion. Note that immediate
deletion of some resources may result in inconsistency or data loss and requires confirmation.
```

Can we add similar phrasing so they understand that resources may be deleted:

```
Force will immediately remove resources from the API and bypasses graceful deletion. Immediate deletion of some resources may result in inconsistency or data loss.
```

Reply:
Great suggestion. Updated.